imap/imapd.c

```diff
@@ -15691,9 +15693,22 @@ static void push_updates(int idling)
                 notify_event_groups->selected.fetchargs.fetchitems) {
                 const char *uidset =
                     json_string_value(json_object_get(msg, "uidset"));
+                struct fetchargs *fa = &notify_event_groups->selected.fetchargs;
 
-                index_fetch(imapd_index, uidset ? uidset : "*", 1,
-                            &notify_event_groups->selected.fetchargs, NULL);
+                /* Open conv.db if we need it */
+                if (fa->fetchitems & (FETCH_MAILBOXIDS|FETCH_MAILBOXES)) {
```

Comment:
Huh, why doesn't this need to open convs for `FETCH_EMAILID|FETCH_THREADID` too?

Reply:
It can get them from the index record.